localstack_services.sh

```diff
@@ -25,33 +25,38 @@ tfswitch 1.6.6
 
 basedir=$(pwd)
 
-printf "${color}=> Cleaning up previous caches, terraform state, and lambda dependencies${reset}\n"
+ACTION=$1
 
-printf "${color}...Purging stale localstack related files${reset}\n"
-find $basedir/env/cloud -type d -name .terragrunt-cache -prune -print -exec rm -rf {} \;
+if [[ "${ACTION}" == "clean" ]]
```

Comment:
Re added this parameter for now. The README file has been updated to explain when to use it. 
Please note that a smarter version of this script is already being tested [here](https://github.com/cds-snc/forms-terraform/pull/626). It will be able to detect whether it should run with or without the clean argument.